packages/flutter_tools/doc/daemon.md

```diff
@@ -233,6 +233,7 @@ Return a list of all available emulators. The `params` field will be a List; eac
 The `launch()` command allows launching an emulator/simulator by its `id`.
 
 - `emulatorId`: the id of an emulator as returned by `getEmulators`.
+- `coldboot`: an optional boolean flag which indicates if the emulator should be cold booted. Only supported for android emulators, silently ignored if emulator type is not android.
```

Comment:
Would this be more consistent as `coldBoot`?

Reply:
Probably, I will change that